docs/source/onnx.md

```diff
@@ -40,11 +40,9 @@ torch.onnx.export(
 )
 ```
 
-Next sections introduce the two versions of the exporter.
-
 ## TorchDynamo-based ONNX Exporter
```

Comment:
Can you update the reference to say `torch.export` based?

Reply:
I thought about this. Do we want to directly use `torch.export.export` (API name) to describe it, or there is a better way?